Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_dateperiod.py

```diff
@@ -147,9 +152,14 @@ def match_simple_cases(self, source: str) -> List[ExtractResult]:
                     if not year_str:
                         year = self.__get_year_from_text(match_year)
 
-                        if not (year >= 1500 and year <= 2000):
+                        if not (year >= 1500 and year <= 2100):
```

Comment:
Ideally these boundaries should reference values defined in the YAML resource files too.

Reply:
Exactly to avoid cases like this where some value was not updated when it changed in another place.